CONTRIBUTING.md

```diff
@@ -24,7 +24,13 @@ There are multiple ways to do this, but basically the end goal is to have a brai
 
 Once you have have found an issue you feel comfortable working on, assign yourself to that issue. This let's others know that you are actively working on it and that they should not work on this ticket.
 
-If this is your first time contributing to open-source or you need a refresher, see this [great guide](https://github.com/firstcontributions/first-contributions#first-contributions) that goes through the steps of contributing to an open-source project.
+***Important: every time you create a new feature branch, start from the latest main branch***
+- Creating a new branch from the main branch:
+  - `git checkout main`
+  - `git pull origin main`
+  - `git checkout -b my-feature-branch main`
+
+If this is your first time contributing to a code or you need a refresher, see this [great guide](https://blog.sashido.io/the-github-flow-tips-and-tricks) that goes through the steps of contributing following the [github flow](https://docs.github.com/en/get-started/quickstart/github-flow).
```

Comment:
if there's a reference to forking that you think might confuse our devs then i'd say we could remove the reference altogether. what do you think?
does the article add anything that we're missing here?
i'm good with whatever you decide

Reply:
This change removes that reference. The article replacing it focuses on the feature -> main work loop instead of the overall topic of contributing to an open-source-project.